spec/db/models.rb

```diff
@@ -36,6 +39,9 @@ class Author < VirtualTotalTestBase
   virtual_sum :sum_recently_published_books_rating, :recently_published_books, :rating
   virtual_delegate :description, :to => :current_photo, :prefix => true, :type => :string
   virtual_delegate :description, :to => :fancy_photo, :prefix => true, :type => :string
+  # delegate to parent relationship
+  virtual_delegate :name, :to => :teacher, :prefix => true, :type => :string
```

Comment:
although this is weird, because now you have author#name but it's actually teacher_name?

Reply:
the fact that you responded and had an EDIT EDIT just further emphasizes that I did not do a good job with the relationship/naming